src/url-fetch/UrlFetchApp.js

```diff
@@ -1,12 +1,15 @@
-const HttpResponse = require('./classes/HttpResponse')
+const UrlFetchAppStubConfiguration = require('./classes/UrlFetchAppStubConfiguration')
 
 class UrlFetchApp {
   static fetch (url, params = {}) {
-    return new HttpResponse()
+    const data = UrlFetchAppStubConfiguration.get(url)
+    if(data)
+      return data.response
+    return null
```

Comment:
This might break existing behavior for users who rely on dummy response instance, can we preserve this functionality and return `new HttpResponse()` instead of `null`, or maybe return `null` only in case UrlFetchAppStubConfiguration has stubs configured?

Reply:
Yes, this can be preserved however there is still a breaking change in this PR. URLFetchApp.fetchAll now accepts an object with 'url' & 'params' as its inner fields. The only way to preserve it from breaking changes is by not passing 'params' & just share url with fetchAll,